pkg/agent/agent_linux.go

```diff
@@ -46,3 +187,96 @@ func (i *Initializer) getTunnelPortLocalIP() net.IP {
 func GetTransportIPNetDeviceByName(ifaceName string, ovsBridgeName string) (*net.IPNet, *net.Interface, error) {
 	return util.GetIPNetDeviceByName(ifaceName)
 }
+
+// saveHostRoutes saves routes which are configured on uplink interface before
+// the interface the configured as the uplink of antrea network.
+// The routes will be restored on OVS bridge interface after the IP configuration
+// is moved to the OVS bridge.
+func (i *Initializer) saveHostRoutes() error {
+	routes, err := netlink.RouteList(nil, netlink.FAMILY_V4)
+	if err != nil {
+		return err
+	}
+	for _, route := range routes {
+		if route.LinkIndex != i.nodeConfig.UplinkNetConfig.Index {
+			continue
+		}
+		if route.Gw == nil {
+			continue
+		}
+		// Skip IPv6 routes before we support IPv6 stack.
+		if route.Gw.To4() == nil {
+			continue
+		}
+		klog.Infof("Got host route: %v", route)
```

Comment:
for making troubleshooting easier, we might want to log also the routes that we skip (noting that we skipped them)